src/MAUI/Maui.Samples/Samples/GraphicsOverlay/CreateAndEditGeometries/CreateAndEditGeometries.xaml.cs

```diff
@@ -0,0 +1,355 @@
+        private void DiscardButton_Click(object sender, EventArgs e)
+        {
+            _geometryEditor.Stop();
+            ResetFromEditingSession();
+        }
+
+        // Remove all graphics from the graphics overlay.
+        private void DeleteAllButton_Click(object sender, EventArgs e)
+        {
+            _graphicsOverlay.Graphics.Clear();
+        }
+
+        private async void MyMapView_GeoViewTapped(object sender, Esri.ArcGISRuntime.Maui.GeoViewInputEventArgs e)
+        {
+            // Return immediately when in an editing session.
+            if (_geometryEditor.IsStarted) return;
+
+            try
+            {
+                // Identify graphics in the graphics overlay using the mouse point.
+                IReadOnlyList<IdentifyGraphicsOverlayResult> results = await MyMapView.IdentifyGraphicsOverlaysAsync(e.Position, 5, false);
+
+                // Try to get the first graphic from the first result.
+                _selectedGraphic = results.FirstOrDefault()?.Graphics?.FirstOrDefault();
+            }
+            catch (Exception ex)
+            {
+                // Report exceptions.
+                await Application.Current.MainPage.DisplayAlert("Error editing", ex.Message, "OK");
+
+                ResetFromEditingSession();
+                return;
+            }
+
+            // Return since no graphic was selected.
+            if (_selectedGraphic == null) return;
+
+            _selectedGraphic.IsSelected = true;
+
+            // Configure the UI depending on the geometry type.
+            GeometryType geometryType = _selectedGraphic.Geometry.GeometryType;
+            if (geometryType == GeometryType.Point || geometryType == GeometryType.Multipoint)
+            {
+                ToolPicker.SelectedIndex = 0;
+            }
+            DisableOtherGeometryButtons(_geometryButtons[geometryType]);
+
+            // Hide the selected graphic and start an editing session with a copy of it.
+            _geometryEditor.Start(_selectedGraphic.Geometry);
+            _selectedGraphic.IsVisible = false;
+        }
+
+        #endregion Edit controls event handlers
+
+        #region Helper methods
+
+        private void CreateInitialGraphics()
+        {
+            // Create symbols for displaying new geometries.
+            _pointSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbolStyle.Square, Color.OrangeRed, 10);
+            _multiPointSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbolStyle.Circle, Color.Yellow, 5);
+            _polylineSymbol = new SimpleLineSymbol(SimpleLineSymbolStyle.Solid, Color.Blue, 2);
+            var polygonLineSymbol = new SimpleLineSymbol(SimpleLineSymbolStyle.Dash, Color.Black, 1);
+            _polygonSymbol = new SimpleFillSymbol(SimpleFillSymbolStyle.Solid, Color.FromArgb(70, 255, 0, 0), polygonLineSymbol);
+
+            // Create geometry objects from JSON formatted strings.
+            var houseCoordinates = (MapPoint)Geometry.FromJson
```

Comment:
I'd pull these json strings out of the method as constant strings, it just tidies up readability. We do this in other samples.